react-intl/index.d.ts

```diff
@@ -227,7 +228,11 @@ declare namespace ReactIntl {
             defaultFormats?: Object;
         }
     }
-    class IntlProvider extends React.Component<IntlProvider.Props, any> { }
+    class IntlProvider extends React.Component<IntlProvider.Props, any> {
+        getChildContext(): {
+            intl: any;
```

Comment:
Well, in this case you can just write it like

```ts
const props = { locale: 'en', messages: localizations.en };
const intlProvider = new ReactIntl.IntlProvider(props, {});

return (intlProvider as any).getChildContext().intl;
```

so types additions make no sense.

I would put here at least `InjectedIntl` for now and when [this](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/15137) PR will land there will be all other properties within `InjectedIntl` type as well.